src/edu/stanford/dlss/wowza/SulWowza.java

```diff
@@ -18,53 +18,128 @@
 public class SulWowza extends ModuleBase
 {
 
-    public static final String DEFAULT_STACKS_TOKEN_VERIFICATION_BASEURL = "http://localhost:3000";
     static String stacksTokenVerificationBaseUrl;
-    /** default setting for stacks service connection timeout (time to establish a connection), in seconds */
-    public static final int DEFAULT_STACKS_CONNECTION_TIMEOUT = 30;
     static int stacksConnectionTimeout;
-    /** default setting for stacks service read timeout (time for reading stream after connection is established),
-     * in seconds */
-    public static final int DEFAULT_STACKS_READ_TIMEOUT = 30;
     static int stacksReadTimeout;
 
+    /** configuration is invalid if the stacks url is malformed */
+    boolean invalidConfiguration = true;
```

Comment:
Did you intend to initialize this to false? It looks like it will always be true?